workitem/expression_compiler.go

```diff
@@ -191,6 +191,17 @@ var DefaultTableJoins = func() TableJoinMap {
 			AllowedColumns:     []string{"name"},
 			ActivateOtherJoins: []string{"space"},
 		},
+		"label": {
+			TableName:  "labels",
+			TableAlias: "lbl",
+			On: Column("lbl", "space_id") + "=" + Column(WorkItemStorage{}.TableName(), "space_id") + `
+		                    AND lbl.id::text IN (
+		                        SELECT
+						jsonb_array_elements_text(` + Column(WorkItemStorage{}.TableName(), "fields") + `->'system.labels')
+					FROM labels)`,
```

Comment:
@baijum I think we should limit this by the space ID as well; otherwise we'll be searching many labels.